src/machine.rs

```diff
@@ -185,6 +185,26 @@ impl<'m> Machine<'m> {
         Ok(machine)
     }
 
+    /// Connect to already running machine.
+    ///
+    /// The machine should be created first via call to `create`.
+    #[instrument]
+    pub async fn connect(config: Config<'m>, pid: i32) -> Result<Machine<'m>, Error> {
```

Comment:
wonder if this needs to return `Result`. It doesn't right now though so we can remove it and add it back later, when/if needed?